### ISSUE TITLE
Gurobi 8.0 test error

### ISSUE THREAD

**Issue by schalil** [JuliaOpt/Gurobi.jl#124]
I installed Gurobi 8.0, added the aliases = ["gurobi80", "gurobi75","gurobi70","gurobi65","gurobi60","gurobi56","gurobi55"]  and used Pkg.test("Gurobi"). I get the following error.

```
Academic license - for non-commercial use only
Optimize a model with 2 rows, 2 columns and 2 nonzeros
Model has 1 quadratic constraint
Coefficient statistics:
  Matrix range     [1e+00, 1e+00]
  QMatrix range    [1e+00, 1e+00]
  Objective range  [0e+00, 0e+00]
  Bounds range     [0e+00, 0e+00]
  RHS range        [1e+00, 2e+00]
Presolve removed 2 rows and 2 columns
Presolve time: 0.00s
Presolve: All rows and columns removed


Barrier solved model in 0 iterations and 0.00 seconds
Optimal objective 0.00000000e+00
SOC3 infeasible: Test Failed
  Expression: MathProgBase.status(m) == :Infeasible
   Evaluated: Suboptimal == Infeasible
Stacktrace:
 [1] macro expansion at C:\Users\schalil\.julia\v0.6\MathProgBase\test\conicinterface.jl:367 [inlined]
 [2] macro expansion at .\test.jl:860 [inlined]
 [3] macro expansion at C:\Users\schalil\.julia\v0.6\MathProgBase\test\conicinterface.jl:357 [inlined]
 [4] macro expansion at .\test.jl:860 [inlined]
 [5] #conicSOCtest#4(::Bool, ::Float64, ::Function, ::Gurobi.GurobiSolver) at C:\Users\schalil\.julia\v0.6\MathProgBase\test\conicinterface.jl:216
Test Summary:                                                                         | Pass  Fail  Total
Testing SOC problems through conic interface with Gurobi.GurobiSolver(nothing, Any[]) |   20     1     21
  SOC1                                                                                |    5            5
  SOC1A                                                                               |    5            5
  SOC2                                                                                |    5            5
  SOC2A                                                                               |    5            5
  SOC3 infeasible                                                                     |          1      1
ERROR: LoadError: LoadError: Some tests did not pass: 20 passed, 1 failed, 0 errored, 0 broken.
while loading C:\Users\schalil\.julia\v0.6\Gurobi\test\mathprog.jl, in expression starting on line 23
while loading C:\Users\schalil\.julia\v0.6\Gurobi\test\runtests.jl, in expression starting on line 19
======================================================================================[ ERROR: Gurobi ]======================================================================================

failed process: Process(`'C:\Users\schalil\AppData\Local\Julia-0.6.2\bin\julia.exe' -Cx86-64 '-JC:\Users\schalil\AppData\Local\Julia-0.6.2\lib\julia\sys.dll' --compile=yes --depwarn=yes --check-bounds=yes --code-coverage=none --color=yes --compilecache=yes 'C:\Users\schalil\.julia\v0.6\Gurobi\test\runtests.jl'`, ProcessExited(1)) [1]

=============================================================================================================================================================================================
**ERROR: Gurobi had test errors**
```

**Comment by blegat** [comment]
This issue is being discussed in https://github.com/JuliaOpt/Gurobi.jl/pull/123

**Comment by odow** [comment]
Closed by #123
